src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -74,7 +82,7 @@ public function parse(string $name, array $config): Filter
             ->setMethod($config['method'])
             ->setStatus($config['status']);
 
-        $this->parseRoute($filter, $config['route']);
+        array_key_exists('route', $config) ? $this->parseRoute($filter, $config['route']) : $this->parseRoutes($filter, $config['routes'] ?? []);
```

Comment:
It's not a good way to use ternary. Ternary MUST be used always for variable assignation, not for strategy.
In fact, you just had an if/else instruction and else MUST be NEVER used.
I really prefer a function to do this or a factory to choose which Parser we need to used depend on the config.